### PR TITLE
6 get/api/articles/article id/comments

### PR DESCRIPTION
**Pull request by LpgLevin**
https://trello.com/b/QY1dVwOs/lily-to-do-list-backend-project

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -306,4 +287,112 @@ describe('/api/articles', function(){
...
+
+                expect(response.body.commentArray[0].article_id).toEqual(9);
+
+            });
+
+        });
+
+        
+    });
+
```
Comment (NRMSMITH) [inline __tests__/app.test.js:365]: Add another 200 test for when an article does not have any comments please :) It should return an empty array!

File: controllers/topics.controller.js [hunk 2]
```diff
@@ -40,3 +40,22 @@ exports.getArticlesInOrder = (request, response, next) => {
...
 
 };
+
+
+exports.getCommentsById = (request, response, next) => {
+
+    const { article_id } = request.params;
+
+    selectCommentsById(article_id)
+    .then((commentArray) => {
```
Comment (NRMSMITH) [inline controllers/topics.controller.js:50]: maybe change this to just 'comments', so then the array is on a key of 'comments' which will help with readability :)

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
Only a couple of changes - one in the controller: change 'commentsArray' to 'comments' for readability (the object will have a key of comments that is an array), and add another test 😄
